components/script/dom/webglrenderingcontext.rs

```diff
@@ -112,6 +112,11 @@ fn has_invalid_blend_constants(arg1: u32, arg2: u32) -> bool {
         (_, _) => false
     }
 }
+
+fn multiply_u8_pixel(a: u8, b: u8) -> u8 {
```

Comment:
I had noticed that, but I didn't imagine that that code would want to grow some sort of format enum for handling different GL formats.